noxfile.py

```diff
@@ -303,89 +196,148 @@ def stop_db(session: Session) -> None:
-            errors.append((path, url, status, details))
+    def parser() -> ArgumentParser:
+        p = ArgumentParser(
+            usage="nox -s exasol-tests -- [-h] [--connector]",
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        )
+        p.add_argument(
+            "--connector",
+            choices=Settings.CONNECTORS,
+            default=Settings.CONNECTORS[0],
+            help="which will be used",
+        )
+        return p
 
-    if errors:
-        session.error(
-            "\n"
-            + "\n".join(f"Url: {e[1]}, File: {e[0]}, Error: {e[3]}" for e in errors)
+    with odbcconfig(Settings.ODBC_DRIVER) as (config, env):
+        args = parser().parse_args(session.posargs)
+        connector = args.connector
+        session.run(
+            "pytest",
+            "--dropfirst",
+            "--db",
+            f"exasol-{connector}",
+            f"{PROJECT_ROOT / 'test' / 'integration' / 'exasol'}",
+            external=True,
+            env=env,
         )
 
 
-@nox.session(name="list-links", python=False)
-def list_links(session: Session) -> None:
-    """List all links within the documentation"""
-    for path, url in _urls(_documentation(PROJECT_ROOT)):
-        session.log(f"Url: {url}, File: {path}")
+@nox.session(name="integration-tests", python=False)
+def integration_tests(session: Session) -> None:
+    """Run integration tests with a specific configuration. For more details append '-- -h'"""
+
+    def parser() -> ArgumentParser:
+        p = ArgumentParser(
+            usage="nox -s integration-tests -- [-h] [--connector] [--db-version]",
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        )
+        p.add_argument(
+            "--connector",
+            choices=Settings.CONNECTORS,
+            default=Settings.CONNECTORS[0],
+            help="which will be used",
+        )
+        p.add_argument(
+            "--db-version",
+            choices=Settings.DB_VERSIONS,
+            default=Settings.DB_VERSIONS[0],
+            help="which will be used",
+        )
+        return p
+
+    args = parser().parse_args(session.posargs)
+    session.notify(
+        find_session_runner(session, "db-start"),
+        posargs=["--db-version", f"{args.db_version}"],
+    )
+    session.notify(
+        find_session_runner(session, f"sqla-tests"),
+        posargs=["--connector", f"{args.connector}"],
+    )
+    session.notify(
+        find_session_runner(session, f"exasol-tests"),
+        posargs=["--connector", f"{args.connector}"],
+    )
+    session.notify(find_session_runner(session, "db-stop"))
+
+
+@nox.session(python=False, name="clean-docs")
+def clean(session: Session) -> None:
+    """Remove all documentation artifacts"""
+    if DOC_BUILD.exists():
+        rmtree(DOC_BUILD.resolve())
+        session.log(f"Removed {DOC_BUILD}")
+
+
+@nox.session(python=False, name="build-docs")
+def build(session: Session) -> None:
```

Comment:
build-docs, build ja not specific enough and could be confused with building then project